lang/octave/README.md

```diff
@@ -0,0 +1,271 @@
+# GNU Octave
+
+![octave-logo](octave-logo.svg)
+
+[GNU Octave](https://www.gnu.org/software/octave/) is a free and open source
+(FOSS) high-level language for array manipulation and linear algebra (very 
+similar to [MATLAB](https://www.mathworks.com/products/matlab.html)). It ships 
+with an IDE and is very good for things like curve-fitting, modeling and 
+parameter optimization, signal processing, and image manipulation.  
+
+In this example, we'll create an Ubuntu 18.04 based container with a basic
+Octave installation. The definition file will allow for easily starting the 
+Octave IDE, or running a script.
+
+## writing the definition file
+
+First, create a directory for your Octave assets. This step is optional, but it 
+will help keep your space tidy.
+
+```
+$ mkdir ~/octave && cd ~/octave
+```
+
+Open an editor (such as `nano` or `vim`) and copy the following contents into a 
+ _definition file_.  In this example we will call it `octave.def` but you can 
+ name it whatever you like.
+
+```
+Bootstrap: library
+From: ubuntu:18.04
+
+%post
+    apt-get update && apt-get -y install octave less libcanberra-gtk3-module
+
+%environment
+    export LC_ALL=C
+
+%runscript
+    octave "$@"
+```
+
+Alternatively, you can obtain the definition file directly from this repo with 
+the following command:
+
+```
+$ wget https://raw.githubusercontent.com/sylabs/examples/master/lang/octave/octave.def
+```
+
+### the definition file explained 
+
+The definition file header instructs Singularity to download a base Ubuntu 18.04
+container from the [Sylabs Container Library](https://cloud.sylabs.io/library).
+
+The `%post` section is executed inside the container at build time.  This is 
```

Comment:
There are two spaces between the end of the first sentence and the start of the next ('...build time.  This is...' )